apps/api/src/app/environments-v1/dtos/environment-response.dto.ts

```diff
@@ -30,6 +31,14 @@ export class EnvironmentResponseDto {
   })
   identifier: string;
 
+  @ApiPropertyOptional({
+    enum: EnvironmentTypeEnum,
+    description: 'Type of the environment',
+    example: EnvironmentTypeEnum.PROD,
+    nullable: true,
+  })
+  type: EnvironmentTypeEnum;
```

Comment:
```suggestion
  @ApiPropertyOptional({
    enum: EnvironmentTypeEnum,
    description: 'Type of the environment',
    example: EnvironmentTypeEnum.PROD,
  })
  type: EnvironmentTypeEnum;
```

Reply:
@djabarovgeorge from what i've i've read in the docs, nullable !== required, where apiproprty optional only handeled the undefined state, nullable is different. and is explicitly added. 

<img width="2216" height="382" alt="Image" src="https://github.com/user-attachments/assets/24004585-d028-4fca-b361-c18254812fea" />